internal/driver/helper.go

```diff
@@ -86,6 +86,7 @@ func randomInt(min, max int64) int64 {
 }
 
 func randomUint(min, max uint64) uint64 {
+	//nolint // SA1019: rand.Seed has been deprecated
```

Comment:
Wasn't able to find a good alternative and the fact that this is a sample device service chose to ignore the lint error.

Reply:
Good enough.

Reply:
I open an issue for that, and we can deal with it in the future
https://github.com/edgexfoundry/device-virtual-go/issues/358